Fabric.Authorization.Client.UnitTests/AuthorizationClientTests.cs

```diff
@@ -1,35 +1,145 @@
+using System;

```

Comment:
What is everyone's thoughts on establishing a naming convention for unit/integration test method names? Below is the convention I've used so far for Identity/Auth (and other companies in the past) that seems to work well:

http://osherove.com/blog/2005/4/3/naming-standards-for-unit-tests.html

Reply:
I can rename these to something of the nature:

<Function name>_<What we are testing>_<Expected result>

That naming convention works fine, but may be long.  Depends on the test runner used, if you need all of that information.  But I am fine with that format.